microsoftgraph/microsoft-graph-explorer-v4
# Task: auto merge dependabot PRs into one PR

## Overview
This workflow merges all dependabot PRs into one then opens one PR against dev

### Demo
<img width="778" alt="image" src="https://user-images.githubusercontent.com/45680252/168841242-62f69d5b-f050-4677-a2d3-0bbc03262b9a.png">

Optional. Screenshots, `curl` examples, etc.

### Notes

- This workflow will be run manually after dependabot has opened all the PRs for the month. 
- The dependabot updates will be merged into the dependabot-upgrades branch, then a new PR against dev is created 
## Testing Instructions

* How to test this PR

- Go to Actions
- Click on CombinePRs action
- Wait for the workflow to run
- Notice that a new PR is created
